javascript/gdx_getting_started_2.html

```diff
@@ -1,3 +1,15 @@
+  <!--
+  This example uses the GDX JavaScript library to collect all of the sensor information from a Vernier GoDirect Sensor. 
```

Comment:
For all of the examples other than getting_started_1, these top two lines of comments seem redundant.